flink-table/flink-sql-gateway/src/main/java/org/apache/flink/table/gateway/service/materializedtable/MaterializedTableManager.java

```diff
@@ -0,0 +1,182 @@
+import org.apache.flink.table.gateway.service.operation.OperationExecutor;
+import org.apache.flink.table.gateway.service.result.ResultFetcher;
+import org.apache.flink.table.gateway.service.utils.SqlExecutionException;
+import org.apache.flink.table.operations.materializedtable.AlterMaterializedTableChangeOperation;
+import org.apache.flink.table.operations.materializedtable.CreateMaterializedTableOperation;
+import org.apache.flink.table.operations.materializedtable.DropMaterializedTableOperation;
+import org.apache.flink.table.operations.materializedtable.MaterializedTableOperation;
+import org.apache.flink.table.refresh.ContinuousRefreshHandler;
+import org.apache.flink.table.refresh.ContinuousRefreshHandlerSerializer;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.apache.flink.api.common.RuntimeExecutionMode.STREAMING;
+import static org.apache.flink.configuration.DeploymentOptions.TARGET;
+import static org.apache.flink.configuration.ExecutionOptions.RUNTIME_MODE;
+import static org.apache.flink.configuration.PipelineOptions.NAME;
+import static org.apache.flink.streaming.api.environment.ExecutionCheckpointingOptions.CHECKPOINTING_INTERVAL;
+import static org.apache.flink.table.api.internal.TableResultInternal.TABLE_RESULT_OK;
+
+/** Manager is responsible for execute the {@link MaterializedTableOperation}. */
+@Internal
+public class MaterializedTableManager {
+
+    private static final Logger LOG = LoggerFactory.getLogger(MaterializedTableManager.class);
+
+    public static ResultFetcher callMaterializedTableOperation(
+            OperationExecutor operationExecutor,
+            OperationHandle handle,
+            MaterializedTableOperation op,
+            String statement) {
+        if (op instanceof CreateMaterializedTableOperation) {
+            return callCreateMaterializedTableOperation(
+                    operationExecutor, handle, (CreateMaterializedTableOperation) op);
+        }
+        throw new SqlExecutionException(
+                String.format(
+                        "Unsupported Operation %s for materialized table.", op.asSummaryString()));
+    }
+
+    private static ResultFetcher callCreateMaterializedTableOperation(
+            OperationExecutor operationExecutor,
+            OperationHandle handle,
+            CreateMaterializedTableOperation createMaterializedTableOperation) {
+        CatalogMaterializedTable materializedTable =
+                createMaterializedTableOperation.getCatalogMaterializedTable();
+        if (CatalogMaterializedTable.RefreshMode.CONTINUOUS == materializedTable.getRefreshMode()) {
+            createMaterializedInContinuousMode(
+                    operationExecutor, handle, createMaterializedTableOperation);
+        } else {
+            throw new SqlExecutionException(
+                    "Only support create materialized table in continuous refresh mode currently.");
```

Comment:
The wording here is a bit strange.  How about change it to .  `Currently, we only support creating materialized tables with continuous refresh mode.`